### ISSUE TITLE
Mushroom biome notes 

### ISSUE THREAD

**Issue by Dylan4ever** [ganymedes01/TheErebus#118]
The mushrooms should differ... Not all directly copy-pasted, but with randomness in height, shape, size, etc. to keep it fresh.

Same with the hollow fallen log and the hollow tree... Height/length, and for the hollow tree the branches, and the shelf shrooms growing on them.

No Redgem should be found in this biome, since Dave is working on his Glowshrooms.

The ''ground'' should be a nice mixture of the new non-growing Dirt, Podzol and Mycellium, in a ratio you see fit.

So, basically, variety!

(this comes from that old thing, should be implemented already:

Humidity: 0.95
Temperature: 0.9

Grass/foliage color:
4E8833)

**Comment by vadis365** [comment]
Isn't all this done already?
